tests/python/TestDescriptors.py

```diff
@@ -785,23 +784,25 @@ def test_findDescByBlobWithLink(self):
         all_queries.append(query)
 
         response, blob_array = db.query(all_queries, [descriptor_blob])
+        self.assertEqual(response[0]["status"], -1)
```

Comment:
Do we still need test_findDescByBlobWithLink since we no longer have capability to link the knn search using _ref? This is basically just a AddDescriptor/AddEntity test now. Or should we change KNN search to search by constraints to leverage defined _ref?

Reply:
Good question, left it in for the moment for us to discuss. It does technically exercise a slightly different path above when its doing its setup, but the search itself is mostly a repeat of the early KNN test.